package.json

```diff
@@ -19,6 +19,7 @@
     "ethereumjs-abi": "0.6.4",
     "ethereumjs-testrpc": "4.1.1",
     "ethereumjs-wallet": "0.6.0",
+    "ethjs": "^0.2.8",
```

Comment:
Can you pin this dependency? Just remove the carrot. If somebody maliciously updates this package in the future to steal private keys we'll get pwned.